lib/ui/router/router_app.dart

```diff
@@ -75,10 +104,40 @@ class HomeRoute extends GoRouteData {
   }
 }
 
+class HomeBranch extends StatefulShellBranchData {
+  static final $navigatorKey = homeNavigatorKey;
+  static const index = 0;
```

Comment:
How about adding a getter that can get the `branchIndex` (instead of `index`) from the `RootTab`? If you do this, I would appreciate it if you could comment that there is a process related to `router_app.dart` (related to `TypedStatefulShellRoute`).

Reply:
Good idea, I implemented and added comment.
[feat: define branchIndex in RootTab and use it in navigation rail and…](https://github.com/FlutterKaigi/conference-app-2023/pull/142/commits/273120cd4f556cfc1e845c0e5a2bac10ea07f96d)